src/components/ActionList/ActionList.scss

```diff
@@ -213,7 +213,7 @@ $active-indicator-width: rem(3px);
     &.active,
     &:active {
       background-image: none;
-      background-color: var(--p-surface-selected);
+      background-color: var(--p-surface-pressed);
       @include recolor-icon(var(--p-interactive));
     }
```

Comment:
We need to break up `.active` and `:active` because `.active` items should look selected.